src/parsers/transactions.ts

```diff
@@ -290,13 +284,54 @@ const overrideTradeRefund = (txn: ZerionTransaction): ZerionTransaction => {
   };
 };
 
-const parseTransaction = (
+const parseTransactionWithEmptyChanges = async (
+  txn: ZerionTransaction,
+  nativeCurrency: string
+) => {
+  const methodName = await getTransactionMethodName(txn);
+  const updatedAsset = {
+    address: ETH_ADDRESS,
+    decimals: 18,
+    name: 'ethereum',
+    symbol: 'ETH',
+  };
+  const priceUnit = 0;
+  const valueUnit = 0;
+  const nativeDisplay = convertRawAmountToNativeDisplay(
+    0,
+    18,
+    priceUnit,
+    nativeCurrency
+  );
+
+  return [
+    {
+      address: ETH_ADDRESS,
+      balance: convertRawAmountToBalance(valueUnit, updatedAsset),
+      description: methodName || 'Contract interaction',
+      from: txn.address_from,
+      hash: `${txn.hash}-${0}`,
+      minedAt: txn.mined_at,
+      name: methodName || 'Contract interaction',
+      native: nativeDisplay,
+      nonce: txn.nonce,
+      pending: false,
+      protocol: txn.protocol,
+      status: TransactionStatus.contract_interaction,
+      symbol: 'contract',
```

Comment:
This is not a blocker but a note for the future: this is the first time we're introducing a symbol of a non-token type. Given that we may have different icons in the future for different types of transactions, we should make this more robust (and rely on "transaction type" instead of a hacked "symbol")

Reply:
i actually did that but couldn't used because that icon was meant for tokens not txs, we could create a parent component that takes care of tokens and other icons https://github.com/rainbow-me/rainbow/pull/2398/files/f1c99d0b73ef5aa605b860cc0d90e5808dbf1550#diff-1df2f9097c6797d911776c84fe906260b0e621ac347c2e81781b1f22b8385e55R324